int64 id_a
int64 id_b
float64 dist
